src/modules/user/services/user.service.ts

```diff
@@ -1,6 +1,37 @@
-import { Injectable } from '@nestjs/common';
+import { BadRequestException, Inject, Injectable } from '@nestjs/common';
+import { Repository } from 'typeorm';
+import type { UserDTO } from '../dto/user.dto';
+import type { UserEntity } from '../entities/user.entity';
+import { UserRepository } from '../repositories/user.repository';
 import { UserService } from './user.service.abstract';
+import { genSalt, hash } from 'bcrypt';
 
 @Injectable()
 export class UserServiceImpl extends UserService {
+
+  constructor(
+        @Inject(UserRepository)
+        private readonly usersRepository: Repository<UserEntity>,
```

Comment:
багато пробілів, залишай лише два після абзацу, в тебе тут шість

```suggestion
    @Inject(UserRepository)
    private readonly usersRepository: Repository<UserEntity>,
```